support/cas-server-support-pm-webflow/src/test/java/org/apereo/cas/pm/web/flow/actions/SendPasswordResetInstructionsActionTests.java

```diff
@@ -71,8 +74,30 @@ public void verifyAction() throws Exception {
             WebUtils.putServiceIntoFlowScope(context, RegisteredServiceTestUtils.getService());
             context.setExternalContext(new ServletExternalContext(new MockServletContext(), request, new MockHttpServletResponse()));
             assertEquals(CasWebflowConstants.TRANSITION_ID_SUCCESS, sendPasswordResetInstructionsAction.execute(context).getId());
+            val tickets = ticketRegistry.getTickets();
+            assertEquals(1, tickets.size());
+            assertInstanceOf(HardTimeoutExpirationPolicy.class, tickets.iterator().next().getExpirationPolicy());
         }
 
+        @Test
+        public void verifyActionMultiUse() throws Exception {
```

Comment:
I realize why you had to write this test this way, but this is not a good, specially because it messes with the spring context cache for tests, and also interferes with parallel processing. 

What you want to do instead is to use nested classes. You'd essentially have 2, where one test class spins up an app context with number-of-uses=1 and the 2nd class is the same thing, except it uses a different property value. 

There are many examples for this pattern; you can use `DefaultCasSSLContextTests` as a baseline, or many others.

Reply:
You're fully right. I did bad. It's fixed now